src/sage/modular/modform_hecketriangle/hecke_triangle_groups.py

```diff
@@ -625,20 +627,20 @@ def get_FD(self, z):
 
         A = ID
         w = z
-        while (abs(w) < ZZ(1) or abs(w.real()) > self.lam()/ZZ(2)):
-            if (abs(w) < ZZ(1)):
+        while abs(w) < ZZ(1) or abs(w.real()) > self.lam()/ZZ(2):
+            if abs(w) < ZZ(1):
```

Comment:
```suggestion
        while abs(w) < ZZ.one() or abs(w.real()) > self.lam() / ZZ(2):
            if abs(w) < ZZ.one():
```
I would actually pull out `ZZ(2)` into a separate variable `two` since it is called so many times in these loops. Although that it is not strictly a PEP8 change...some similar type of micro-optimization changes are made elsewhere...